src/app/app-routing.module.ts

```diff
@@ -117,7 +120,9 @@ const routesMob: Routes = [
   {
     path: "search-results/:id:breadcrumb/occurrences/texts/source/:id:breadcrumb",
     component: GlossaryArticleSourceComponent
-  }
+  },
+  { path: "404", component: PageNotFoundComponent },
+  { path: "**", redirectTo: "404" }
```

Comment:
Unrelated to this PR, but do we need to define the mobile routes separately? They look pretty much the same, so would it be better to share a base value? The main difference seems to be some repetition for children of `/search/` in the mobile routes, but maybe that's not needed anyway?

Unless I'm missing something! This can happen in a separate PR, anyway.